src/react_native/utils.cljs

```diff
@@ -37,3 +39,18 @@
         props       (when (map? first-child) first-child)
         children    (if props (rest argv) argv)]
     [props children]))
+
+(defn convert-prop-value
+  [x]
+  (cond
+    (reagent.util/js-val? x) x
+    (reagent.util/named? x)  (name x)
+    (map? x)                 (reduce-kv template/kv-conv #js {} x)
+    (vector? x)              (to-array (mapv convert-prop-value x))
+    (coll? x)                (clj->js x)
+    (ifn? x)                 (fn [& args]
+                               (apply x args))
+    :else                    (clj->js x)))
+
+;; Needed to transform component's prop names in vectors, as supported by React Native.
```

Comment:
I think it would be beneficial for reader to see a more detailed comment here about how it works and what `template/convert-prop-value` does. 
Also the docstring for `convert-prop-value` could help like it did in `utils.transforms/styles-with-vectors` before.
Wdyt?

Reply:
Sure! @vkjr  I'll improve the explanation in this part!